### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by smith-and-web** [HvyIndustries/crane#135]
Parsing never finishes. Running on Ubuntu 16.04 and the Insiders Build.

```
[INFO] Checking the current version of Crane

[INFO] Watching these files: {**/*.php}

[INFO] Preparing to parse 7386 PHP source files...

[ommitted]

[ERROR] There was a problem parsing PHP file: ...SabreAMF/ByteArray.php

[ERROR] [TypeError: Cannot read property '0' of null]

...

[ERROR] There was a problem parsing PHP file: ...tcpdf/tcpdf_parser.php

[ERROR] [TypeError: Cannot read property '3' of undefined]

...
```

Eventually just stalls on a file which is different every time. When doing a reindex:

```
[INFO] Re-indexing PHP files in the workspace...

[INFO] Rebuilding the project files

[INFO] Preparing to parse 7386 PHP source files...
```

Then hangs again. Usually gets ~4% done.

**Comment by nevadascout** [comment]
Hi, thanks for reporting this and sorry you're having an issue.

I need to try and figure out which line of code is causing the parser to break, so are you able to share the source code of one of the files that is breaking with me? (eg. the `SabreAMF/ByteArray.php` or `tcpdf/tcpdf_parser.php` file?)

**Comment by nevadascout** [comment]
This should be fixed with the php-parser library updated to 2.0.0

**Comment by nevadascout** [comment]
Fixed in v0.3.0
